gen/p2.hoon

```diff
@@ -0,0 +1,3200 @@
+::
```

Comment:
Is the logic in this file different from in hoon.hoon?

Reply:
@benjamin-tlon, bumping this question. Should `+p2` still be included?